ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13-periodics.yaml

```diff
@@ -37044,7 +37141,11 @@ periodics:
       job_states_to_report:
       - failure
       - error
-      report_template: Job {{.Spec.Job}} failed. <{{.Status.URL}}|View logs>
+      - success
```

Comment:
Noticed that some jobs report `success` state, while not for the other jobs. Is it expected ?

Reply:
@liangxia Yes, it is our expect status.